tests/std/tests/VSO_0180469_ptr_cat/test.cpp

```diff
@@ -305,13 +391,16 @@ void equal_safe_test_cases() {
     // No user-defined types
     test_case_Equal_memcmp_is_safe<false, base_class, base_class>();
 
+#ifdef __cpp_lib_byte
+    test_case_Equal_memcmp_is_safe<true, byte, byte>();

```

Comment:
Nitpick: Should we add some failure test, e.g `test_case_Equal_memcmp_is_safe<false, char, byte>();`

Reply:
Excellent idea! I'll push that.